src/tvu.rs

```diff
@@ -327,6 +326,9 @@ pub mod tests {
         let (genesis_block, mint_keypair) = GenesisBlock::new(starting_balance);
         let tvu_addr = target1.info.tvu;
         let bank = Arc::new(Bank::new(&genesis_block));
+        // 2 tokens are consumed by the genesis
+        let starting_balance = starting_balance - 2;
+        assert_eq!(bank.get_balance(&mint_keypair.pubkey()), starting_balance);
```

Comment:
I would just put as `let starting_balance = bank.get_balance(..)` otherwise if genesis needs 3 tokens tomorrow, all this stuff has to be updated.

Reply:
Fair, there's a bunch of tests that are doing this now. I'm working on a PR on the side that just gets rid of this altogether.

Reply:
Ok, no problem. Fine to leave like this for now.